README.md

```diff
@@ -1 +1,75 @@
-# to-do-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+## Built With
+
+- Html
+- Css
+- JavaScript
+- Webpack
+
+
+## Live Demo (if available)
+  https://jahongiry.github.io/to-do-app/
+
+## Screenshots (if available)
+
+
+## Getting Started
+
+To get a copy of this project running on your local computer, do the followings:
+
+### Prerequisites
+
+- Must have a working computer
+- A Web-Browser installed (Chrome - Recommended)
+
+### Setup
+
+- Install _VSCode_ ([link](https://code.visualstudio.com/download))
+- Clone this project repository on your local computer. ([link](https://github.com/jahongiry/to-do-app.git))
+- Open the project directory with _VSCode_
+
```

Comment:
### OPTIONAL
- [ ] Kindly add commands to get this project running  locally; `npm install` to install  and `npm run dev` to run the server 👍🏿